bundles/basic_modes/mode_definitions.moon

```diff
@@ -299,9 +299,9 @@ common_auto_pairs = {
     comment_syntax: '#'
     indentation: {
       more_after: {
-        { ':%s*$', 'else:%s*$' }
+        { ':%s*$', 'else%s*:%s*$' }
```

Comment:
So with this latest change it seems we're almost back to the way it used to be, except we're allowing some space before the colon. Is this what you intended?